### ISSUE TITLE
some changes of the hit rate plot?

### ISSUE THREAD

**Issue by LilySnow** [DeepRank/deeprank#31]
1. Can we plot hit rate curves from different epochs on one plot? 
2. Could we also plot hit rate up to top N, where N is the number of decoys for this test case? 
3. Can we also plot HADDOCK score on the hit rate plot so that we can compare with haddock? Haddock scores can be extracted from the pdb files of models. I have also extracted them and put them here: @alembick:/home/lixue/DBs/BM4/haddockScore/water

**Comment by NicoRenaud** [comment]
1. I modified the code so that the hit plot data can be exported at each epoch in the hdf5 output file (still have to push it). I've modified DeepXplorer so that we can plot the hit rate either for individual epoch or multiple epoch on one graph. 

2. That's what's done already no ?

3. We'll have to somehow include the HADDOCK score and corresponding hit plot to the output hdf5 file Another solution would be to have an external script that reads the HDF5 file and the Haddock score and plot all that.

**Comment by NicoRenaud** [comment]
there is now a new option to save all the epoch in the output hdf5 file

```pyhton
model.train(....,save_epoch='all')
```

Then in the .hdf5 file there will be one group for each epoch. In this group there are the train/valid/test subgroup and each subgroup will contain the hit rate data.

You can vizualize all that with the DeepXplorer. You can select mulitple epoch group and right click. You'll have the option to plot the hit plot for the seleted epochs. I've also uploaded a hdf5 file containing the haddock score. If you also select this one with the epoch it will also plot the hit rate of the haddock score.

**Comment by NicoRenaud** [comment]
But in that case we will only see a portion of the trainging and validatiton plot no ?
We could have either:
    a percentage of the x axis so that training/validatation/test have the same x axis
    individual plot for the training/validatation/test cases
What do you think ?
